soteria-rust/lib/dune

```diff
@@ -14,6 +14,9 @@
   iter
   zarith
   hashcons
+  grace
+  grace.ansi_renderer
+  spectrum
```

Comment:
Is it really worth importing new library here? Fmt already has style formatting. Not as easy to write as spectrum but it might be worth trying to keep dependencies under control?

Reply:
```ocaml
let pp_string_red = Fmt.styled (`Fg `Red) Fmt.string
```
You don't have syntactic beauty of `@{<red> ...}` but same power